### PR TITLE
check window width for navbar and hold in state to re-render as needed

### PR DESCRIPTION
**Pull request by cjhaddad**
#248  💅🏼 What issue does this fix? fix nav bar responsiveness to use hamburger menu at <= 768px

## 🍂 Before You Submit

* [ ] Did you write tests?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [ ] Did you run `yarn lint` in `/client`?
* [ ] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

### CODE REVIEW

File: client/src/_shared/LoggedInLayout.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const { useBreakpoint } = Grid
 export function LoggedInLayout({ children, title }) {
   const dispatch = useDispatch()
   const { t, i18n } = useTranslation()
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth)
```
Comment (katelovescode) [inline client/src/_shared/LoggedInLayout.js:17]: 😭  Ant Design, why you make us do this

File: client/src/_shared/LoggedInLayout.js [hunk 2]
```diff
@@ -24,28 +26,55 @@ export function LoggedInLayout({ children, title }) {
...
+
+    return (
+      <Dropdown overlay={mobileMenu}>
+        <MenuOutlined />
+      </Dropdown>
+    )
+  }
+
+  useEffect(() => {
+    window.addEventListener('resize', setWidth)
```
Comment (katelovescode) [inline client/src/_shared/LoggedInLayout.js:74]: 😭  Here, let me reimplement browsers 😭

**Review by katelovescode — APPROVED** [review]
This looks great, Candice, even though it sucks we have to do it this way.  Nice work.  Do you mind adding a comment above the useEffect explaining why we had to do this for later maintenance record?

File: client/src/_shared/LoggedInLayout.js (hunk 1 above)
Comment (cjhaddad) [reply client/src/_shared/LoggedInLayout.js:17]: lol I know...it makes sense what they are doing with their breakpoints, bc they're going with min-width, but I wish that screens object was a little different. sigh

**Comment by cjhaddad** [comment]
> This looks great, Candice, even though it sucks we have to do it this way. Nice work. Do you mind adding a comment above the useEffect explaining why we had to do this for later maintenance record?

Will do!

**Pull request merged**